.github/actions/send-slack-notification/action.yml

```diff
@@ -79,6 +79,7 @@ runs:
 
     - name: Send slack notification
       id: slack
+      if: ${{ inputs.token != '' }}
```

Comment:
Well if the token is empty we'd better throw an error, not skip

Reply:
and would be better a separate pr if this deviate from the main purpose of this one